src/styles/blocks/button-width.scss

```diff
@@ -0,0 +1,23 @@
+.button-width {
+  margin: 0;
+
+  &__style {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+
+    width: 280px;
+    height: 56px;
+    background-color: #d12d35;
+    border: #d12d35;
```

Comment:
The 'border' property is missing a value. If you meant to set a border color, use 'border-color' instead. If you meant to set a border width and style as well, please add those values.